### ISSUE TITLE
Error of database module in executable file.

### ISSUE THREAD

**Issue by mm-ansarian** [mm-ansarian/Capital_Management#1]
Hello.
There is a problem with the executable file that dosn't allow the application to run.

I've completed the codes and I tested it in my IDE, Everything works correctly in the IDE;
But when I convert the `app.py` to an executable file using pyinstaller, and try to open it, this error occurs:

<img width="298" alt="Problem" src="https://github.com/user-attachments/assets/3c1e291f-bea1-474d-a04f-2a3acd8fbcf2" />

The error which occurs when opening the .exe file.

It seems that `mysql-connector-python` is not defined for the executable file and it can't import this module, But in the IDE, there is no problem!

I checked everything, but I coudn't finde any problem.
I tried lots of ways to handle this, but noway worked.
I used hidden-import, I checked the installed packages on the virtual environment that was running pyinstaller and ...
But the problem didn't fix.

In short, I convert the `app.py` to an executable file but when I try to open it, the executable file can not import the database module and the program stops by the `ModuleNotFoundError`.

If you can fix this problem, please announce me.
